packages/python/plotly/plotly/tests/test_optional/test_px/test_px_input.py

```diff
@@ -138,6 +140,64 @@ def test_name_heuristics():
     assert fig.data[0].hovertemplate == "y=%{marker.size}<br>x=%{y}<extra></extra>"
 
 
+def test_performance_b64():
+    rand_arr_1 = np.random.random(100000)
+    rand_arr_2 = np.random.random(100000)
+    b64_arr_1 = b64(rand_arr_1)
+    b64_arr_2 = b64(rand_arr_2)
+
+    # Test the performance of the base64 arrays
+    b64_start = time.time()
+    df_b64 = pd.DataFrame(dict(x=b64_arr_1, y=b64_arr_2))
+    fig = px.scatter(df_b64, x="x", y="y", width=800, height=800)
```

Comment:
In this case it might be better to use `go.Figure(data=go.Scatter(...))` instead of `px.scatter`.